shell/src/main/java/alluxio/cli/fsadmin/report/UfsCommand.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fsadmin.report;
+
+import alluxio.client.file.FileSystemMasterClient;
+import alluxio.util.FormatUtils;
+import alluxio.wire.MountPointInfo;
+
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * Prints under filesystem information.
+ */
+public class UfsCommand {
```

Comment:
It's strange that this and `MountCommand` do the same thing but have different names. Should we just call this `MountCommand` and make it a wrapper around the fs shell `MountCommand`? @apc999 what are your thoughts on the organization?

Reply:
After merging the operation command in another PR https://github.com/Alluxio/alluxio/pull/7010, I consider adding ufs session count information in this ufsCommand. 

The operation information from metrics system contains ufs session count of each mounted url.